test/functional/detailed_guides_controller_test.rb

```diff
@@ -50,20 +50,6 @@ class DetailedGuidesControllerTest < ActionController::TestCase
     end
   end
 
-  view_test "show includes any links to related mainstream content" do
```

Comment:
They assert that the HTML has both a link and a title for a related mainstream content. My understanding is the title only gets unpacked in government-frontend so Whitehall shouldn't expect to have it? Or is that not the case?